### PR TITLE
Improve pgkdown References

### PR DESCRIPTION
**Pull request by lorenzwalthert**
Some docs

### CODE REVIEW

**Comment by lorenzwalthert** [comment]
@assignUser please have a look if the grouping makes sense for you.

File: _pkgdown.yml [hunk 1]
```diff
@@ -12,3 +12,42 @@ articles:
...
+  - refs_install
+  - benchmark_run_ref
+  - benchmarks_analyze
+- title: "Pinning"
+  desc: >
+    Working with assets accross branches.
+- contents:
+  - pin_assets
+  - path_pinned_asset
+- title: "Non-functional documentation"
```
Comment (assignUser) [inline _pkgdown.yml:38]: Should it not be Non-function?

**Review by assignUser — APPROVED** [review]
Yes, I think these make sense!

File: _pkgdown.yml (hunk 1 above)
Comment (lorenzwalthert) [reply _pkgdown.yml:38]:
```suggestion
- title: "Non-function documentation"
```

**Pull request merged**